misaligned.py

```diff
@@ -1,13 +1,39 @@
+from wire_color_pairs.even_color_pairs import EvenColorPairs
 
-def print_color_map():
-    major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
-    minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
+major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
+minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
+
+
+def generate_color_map():
+    color_map = []
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
-            print(f'{i * 5 + j} | {major} | {minor}')
-    return len(major_colors) * len(minor_colors)
+            color_map.append(((i * 5 + j), major, minor))
+    return color_map
+
+
+def get_color_pair_from_number(pairnumber):
+    color_pair = generate_color_map()[pairnumber]
+    return color_pair[1:]
+
+
+def format_color_map(color_map):
+    formatted_color_pairs = ""
+    for color_pair in color_map:
+        formatted_color_pairs += " | ".join([str(item) for item in color_pair]) + "\n"
+    return formatted_color_pairs
+
+
+def print_color_map(color_map):
+    print(format_color_map(color_map))
+
+
+expected_color_map = EvenColorPairs()
+actual_even_color_map = generate_color_map()
+assert get_color_pair_from_number(1) == expected_color_map.get_color_from_pairnumber(1)

```

Comment:
expected_color_map is 'another' implementation of printing the reference manual.
This is definitely a good way to test. Alternative to consider: Would it be easier to state the properties of a reference manual and assert on it? Instead of another implementation?

Reply:
Agreed!

_I simply did another implementation just as an improvement as the previous assignment and this is similar _